data/changelog.txt

```diff
@@ -72,6 +76,23 @@ Adds new skateboard vehicle
 Most scenarios now provide vision of the nearest city on start
 Alternative resolution to clean back bay quest
 Add a synagogue
+Hub 01 robots and gear
+Adds a mutable stream to the mapgen
+Adds chat topics to the futile fighter and overconfident officer
+Slightly mutated dog variants
+New unique overmap special with associated one-time dungeon event.
+[Innawoods] Stream mapgen followup
+stun gun bionic
+Remove not used `welding component kit` item
+Updated the description for Chelsea Boots to use the correct usage of 'donning' vs. the incorrect 'dawning'
+Make bone broth better by extracting more calcium using acified water
+adding nylon quivers with MOLLE capabilities
+house roadmap: update house_01 in-construction variant to use modern code
+Miscellaneous recipe category fixes/additions
+More zombie harvest snippets
+Solar Farm
+Two new civilians: Parent and hospitalized varieties
+Add Electric Masonry Saw
```

Comment:
```suggestion
Adds a mutable stream to the mapgen
```

Reply:
I think the mutable stream is Innawoods only?

Reply:
No problem, in that case I think the line should be moved over to the mods section